src/styles/Grid/Flex.js

```diff
@@ -24,6 +24,10 @@ const FlexColumnCenter = styled(FlexCenter)`
 const FlexColumn = styled.div`
   display: flex;
   flex-direction: column;
+
+  .fpyAse {
```

Comment:
does this do anything? :D

Reply:
Yeah the card in the FlexColumn has a marginTop, since it was used in many places(like onboarding i think), I didn't want to cause some layout issues that's way I added that 👆